system-apps/desktop-documentation-viewer/webClient/src/App-css.js

```diff
@@ -0,0 +1,174 @@
+
+
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+var styles = {
+  iframeFont :{
+    fontFamily: "sans-serif",
+    fontSize: "0.8em"
+  },
+
+  boldText :{
+    fontWeight: "600"
+  },
+
+  biggerBoldText :{
+    fontSize: "1.1em",
+    fontWeight: "600"
+  },
+
+  divInput :{
+    marginTop: "5px",
+    marginBottom: "10px",
+    paddingTop: "5px",
+    paddingBottom: "5px",
+    width: "300px"
+  },
+
+  divTextareaInput :{
+    marginTop: "5px",
+    marginBottom: "5px",
+    paddingTop: "5px",
+    paddingBottom: "5px",
+    width: "95%"
+  },
+
+  iframeInput :{
+    width: "100%",
+    paddingLeft: "5px"
+  },
+
+  inputCorner :{
+    borderRadius: "4px"
+  },
+
+  inputHeight :{
+    height: "20px"
+  },
+
+  inputText :{
+  },
+
+  iframeButton :{
+    borderRadius: "4px",
+    margin: "10px 10px 10px 10px",
+    fontWeight: "700"
+  },
+
+  rightAlign :{
+    textAlign: "right",
+    paddingRight: "50px"
+  },
+
+  shadowed :{
+    boxShadow: "3px 3px 10px"
+  },
+
+  hideIt :{
+    display: "none"
+  },
+
+  bottom10 :{
+    marginBottom: "10px"
+  },
+
+  displayText :{
+    borderRadius: "4px",
+    border: "1px solid grey",
+    backgroundColor: "white",
+    height: "18px",
+    width: "96%",
+    marginTop: "3px",
+    fontSize: "small"
+  },
+
+  disableEffect :{
+    color: "grey"
+  },
+
+  h1 :{
+    textAlign: "center"
+  },
+
+  serverResponse :{
+    width: "100%",
+    height: "180px"
+  },
+
+  testPanel :{
+    padding: "5px 5px 5px 5px"
+  },
+
+  testPanelContainer :{
+    flexDirection: "row",
+    display: "flex"
+  },
+
+  pluginTestPanel :{
+    borderRight: "outset",
+    flexGrow: "1"
+  },
+
+  dataserviceTestPanel :{
+    borderLeft: "outset"
+  },
+
+  paddingRight8 :{
+    paddingRight: "8px"
+  },
+
+  paddingLeft3 :{
+    paddingLeft: "3px"
+  },
+
+  App :{
+    textAlign: "center"
+  },
+
+  AppLogo :{
+    height: "40vmin",
+    pointerEvents: "none",
+  },
+
+  AppHeader :{
+    backgroundColor: "#282c34",
+    minHeight: "5vh",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    fontSize: "calc(10px + 2vmin)",
+    color: "white",
+  },
+
+  AppIntro : {
+    textAlign: "center",
+  },
+
+  AppLink :{
+    color: "#61dafb",
+  },
+
+  body :{
+    margin: "0",
+  }
+};
```

Comment:
I don't remember a lot about how React handles CSS, but these generic CSS classes look like accidents waiting to happen in terms of styles bleeding into each other. The names are generic and if possible, should be renamed for their purpose. 

If this isn't possible (for example, if you're overriding some CSS for a library component, but it doesn't look to me like you're using a UI library?) you could try some CSS local variable hackery available in React
https://speakerdeck.com/vjeux/react-css-in-js?slide=7

Otherwise, lmk if I'm forgetting some React fundamentals and these are all already local by default